docs-src/go/how-to-list-workflow-executions-using-the-client-in-go.md

```diff
@@ -9,4 +9,70 @@ tags:
   - client
 ---
 
-Use [`Client.ListWorkflow`](https://pkg.go.dev/go.temporal.io/sdk/client#Client.ListWorkflow).
+There are several functions for listing [Workflow Executions](/concepts/what-is-a-workflow-execution) available on the [Temporal Go SDK](https://pkg.go.dev/go.temporal.io/sdk).
+Each function retrieves Workflow Executions according to a given [Query](/concepts/what-is-a-query) or request filters.
+
+To list all Workflow Executions, use [`ListWorkflow`](https://pkg.go.dev/go.temporal.io/sdk/client#Client.ListWorkflow) on the [Temporal Client](/concepts/what-is-a-temporal-client).
+
+```go
+resp, err := temporalClient.ListWorkflow(ctx.Background(), req)
+if err != nil {
+  return err
+}
+
+fmt.Println("First page of results:")
+for _, exec := range resp.Executions {
+  fmt.Println("Workflow ID %v\n", exec.Execution.WorkflowId)
+  fmt.Println("Next page of results:")
+}
+```
+
+To return a list of archived Workflow Executions, use `ListArchivedWorkflow()` on the Client.
+Make sure that the Temporal Cluster or target [Namespace](/concepts/what-is-a-namespace) is configured for [visibility archival](/concepts/what-is-standard-visibility) before using this function.
```

Comment:
```suggestion
Make sure that the Temporal Cluster or target [Namespace](/concepts/what-is-a-namespace) is configured for [Archival](/concepts/what-is-archival) before using this function.
```
Is this change accurate? (I could also make a case for linking to https://temporal-documentation-ao26cn23g.preview.thundergun.io/cluster-deployment-guide#set-up-archival, but that information is only one click away from the definition of Archival.)